src/Backend/Modules/Blog/Layout/Templates/EditCategory.html.twig

```diff
@@ -63,7 +63,7 @@
           </button>
         </div>
       </div>
-      {% if showBlogDeleteCategory %}
+      {% if showBlogDeleteCategory and allowBlogDeleteCategory %}
```

Comment:
isn't the showBlogDeleteCategory variable already checked in the BackendBlogModel?

please read the code very well when making changes like this